src/examples/src/widgets/list/Draggable.tsx

```diff
@@ -0,0 +1,52 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import List, { ListOption } from '@dojo/widgets/list';
+import icache from '@dojo/framework/core/middleware/icache';
+import Example from '../../Example';
+import {
+	createMemoryResourceTemplate,
+	createResourceMiddleware
+} from '@dojo/framework/core/middleware/resources';
+
+const resource = createResourceMiddleware();
+const factory = create({ icache, resource });
+
+const animals = [
+	{ value: 'cat' },
```

Comment:
Could you remove the repetition from these values please so that the active states etc still work correctly.
It would also be good to demo the same functionality with a paginated list, perhaps generate a larger data set.